db/migrate/20221213215817_create_sightings.rb

```diff
@@ -0,0 +1,12 @@
+class CreateSightings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :sightings do |t|
+      t.string :longitude
+      t.string :latitude
+      t.string :date
```

Comment:
Date can be used as a string, but consider using the date data type for Active Record.